Cloud_Community_REST_API_Docs.md

```
@@ -0,0 +1,1065 @@
+        "code": "NOT FOUND",
+        "message": "요청한 리소스를 찾을 수 없습니다.",
+        "details": {}
+    }
+}
+```
+
+---
+
+### 내가 쓴 게시글 목록
+
+**GET** `/users/me/posts` 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)